.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,80 @@
+
+Provide a general summary of your changes in the title above.
+
+Fill in the below **Description** section with minimal text describing the changes/new contributions in this PR and replace `<...>` as appropriate.
+
+Remove the beginning `<!--` and ending `-->` comment markers for the correct **checklist** to be used for this PR and delete the other checklist.  (The checklist items for bssw.io site contributions are shown by default.  The checklist for internal files not displayed in the bssw.io site is given below that.)
+
+Any checklist items that don't apply can be striked out by adding `~~` to the beginning and end of the checklist item as `* ~~[] <checklist-item>~~`.  Also, remove the strikeout markers `~~` for the [wikize_refs.py] checklist items if using formal citations for bssw.io contributions.
+
+
+# Description
+
+EB Member: @`<eb-member-id>`
+
+Addresses issue #`<issue-id>`
+
+`<Other minimal information about the PR.>`
+
+
+## PR checklist for files displayed on bssw.io site
+
+* [ ] `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
+* [ ] Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
+* [ ] Assign this PR to the EB member `<eb-member-id>`.
+* [ ] Assign this PR to the author of the PR `<pr-author-id>`.
+* [ ] Add label `content: <content-type>` for the type of contribution.
+* [ ] Add to Project `Content Development` (see [Content Development]).
+* [ ] Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
+* [ ] Add one or more Reviewers.
+* [ ] Add [meta-data] to the `*.md` file(s) (set `Publish: preview`).
+* ~~[ ] Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
+* [ ] Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
+* [ ] Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
+* [ ] Make any final changes to the PR based on feedback.
+* ~~[ ] Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
+* [ ] Rebuild [preview] site and re-confirm content looks correct.
+* [ ] Ensure at least one reviewer signs off on the final changes.
+* [ ] Change meta-data to `Publish: yes` and commit if fully ready to publish.
+* [ ] Move the PR to "Ready to Publish" in [Content Development].
+* [ ] Assign to one of the site maintainers to carry out final publication steps.
+* [ ] Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
+* [ ] Merge PR. Should automatically move to "Done" in [Content Development].
+* [ ] Verify new new contribution shows up on [bssw.io] as expected.
+
+NOTE: Optional checklist items can be striked out with adding `~~` to beginning and end of the checklist item as `* ~~[ ] <checklist-item>~~`.
```

Comment:
Ok so it looks like the checklist is setup to *default* strikeout those items related to citations/refs. That seems like a good idea because that use case is less common.

The note at the bottom then confuses me a bit because it describes how to *stikeout* (more) optional items and not how to **UN**strikeout items that may need to apply (e.g. for cases where citations/refs are used).

Reply:
> The note at the bottom then confuses me a bit because it describes how to stikeout (more) optional items and not how to UNstrikeout items that may need to apply (e.g. for cases where citations/refs are used).

@markcmiller86, the instructions above mention removing strikeout markers for the wikize_refs.py item but it is worth mentioning removing strikeout markers in this below note as well.  I can change that sentence to mention adding and remove strikeout markers `~~`.  I will post that as a new PR so we can iterate more on this checklist now that we have something in place.

Reply:
@markcmiller86, I posted PR #771 with that change.  Please provide comments in that PR #771 for any other thing you might see that needs tweaked.